patches/server/0919-Add-api-for-disabling-conduits-attacking-entities.patch

```diff
@@ -0,0 +1,73 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: u9g <git@u9g.dev>
+Date: Mon, 27 Jun 2022 14:02:35 -0400
+Subject: [PATCH] Add api for disabling conduits attacking entities
+
+
+diff --git a/src/main/java/net/minecraft/world/level/block/entity/ConduitBlockEntity.java b/src/main/java/net/minecraft/world/level/block/entity/ConduitBlockEntity.java
+index 05eab04e4aec4151018f25b59f92ddbbb4c09f87..51a1dc233df7da7af0d5841ad76407684a3ec5f9 100644
+--- a/src/main/java/net/minecraft/world/level/block/entity/ConduitBlockEntity.java
++++ b/src/main/java/net/minecraft/world/level/block/entity/ConduitBlockEntity.java
+@@ -51,6 +51,7 @@ public class ConduitBlockEntity extends BlockEntity {
+     @Nullable
+     private UUID destroyTargetUUID;
+     private long nextAmbientSoundActivation;
++    private boolean canAttackEntities = true;
+ 
+     public ConduitBlockEntity(BlockPos pos, BlockState state) {
+         super(BlockEntityType.CONDUIT, pos, state);
+@@ -64,6 +65,7 @@ public class ConduitBlockEntity extends BlockEntity {
+         } else {
+             this.destroyTargetUUID = null;
+         }
++        canAttackEntities = nbt.hasUUID("Paper.CanAttackEntities") && nbt.getBoolean("Paper.CanAttackEntities"); // Paper
```

Comment:
not a UUID so I doubt this would work.
This also seems to default to false if the tag doesn't exist? That doesn't seem right

Also, this doesn't seem to be saved anywhere?